### ISSUE TITLE
When clicking on the link, the accordion opens completely.

### ISSUE THREAD

**Issue by ysknsid25** [ohbarye/goofi#732]
Before Click

![image](https://user-images.githubusercontent.com/44870505/236624403-19b769e8-1201-4ff0-8425-0a83f2298b0f.png)

After Click

![image](https://user-images.githubusercontent.com/44870505/236624437-b3bac98b-fa67-4a3f-bbf3-80b6fff4bb3b.png)

**Comment by ohbarye** [comment]
Could you elaborate on what's the problem with that?

**Comment by ysknsid25** [comment]
My apologies for my poor explanation.
The behavior is as follows.

https://user-images.githubusercontent.com/44870505/236657409-cfd536f3-7f97-4555-a3db-09bb37ff071a.mp4

I was uncomfortable with the unintentional opening of the accordion when I returned to the original screen. (Shouldn't it be left open? (I mean, it should not be left open.)

However, since everyone feels differently, if the product owner is not bothered by it, there is no need to modify it.

**Comment by ohbarye** [comment]
Ah, I now understand what you're saying.
Either behavior is acceptable to me. If you feel awkward about the current behavior, it's okay to open a pull request to change it.

**Comment by ysknsid25** [comment]
Yes,sir. I will look into it and if I can respond right away, I will send a pull request later🫡

**Comment by annanya-mathur** [comment]
I would like to work over this issue

**Comment by ysknsid25** [comment]
@annanya-mathur 

Thank you!
Please solve it!

**Comment by IgorSilva0** [comment]
Hi @ysknsid25 ,

I would like to work on this enchantment/issue
